app/components/history_version_component.html.erb

```diff
@@ -112,6 +112,26 @@
                         <dd class="text-md p-1 break-words"><%= v %></dd>
                       </div>
                     <% end %>
+
+                    <% keys_no_longer_existent =
+                      prev_changes_value_hash.keys - changes_from_prev_version_hash.keys %>
+
+                    <% if keys_no_longer_existent.length.positive? %>
+                      <div class="col-span-3">
+                        <p class="text-red-500 dark:text-red-400"><%= t(:"components.history_version.keys_deleted") %></p>
```

Comment:
**nitpick:** You could, and I have done this, [create a translation file next to the component file](https://viewcomponent.org/guide/translations.html).

Reply:
As discussed this will possibly be done later if we refactor this into smaller components. Don't want to have translation files directly under the `components/` directory as this is where the history component and history version components live and this is where the translations files would need to live as well